packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -1,63 +1,79 @@
 import * as React from 'react';
 import * as renderer from 'react-test-renderer';
-import { mount } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
 import { SearchBox } from './SearchBox';
 import { KeyCodes } from '../../Utilities';
+import { ISearchBoxProps } from './SearchBox.types';
+import { ISearchBoxState, SearchBoxBase } from './SearchBox.base';
 
 // tslint:disable:jsx-no-lambda
 
 describe('SearchBox', () => {
+  let component: renderer.ReactTestRenderer | undefined;
+  let wrapper: ReactWrapper<ISearchBoxProps, ISearchBoxState, SearchBoxBase> | undefined;
+
+  afterEach(() => {
+    if (component) {
+      component.unmount();
```

Comment:
Commenting as an FYI that you _may_ find the test utilities introduced in the following PR useful for handling auto-unmounting -- https://github.com/OfficeDev/office-ui-fabric-react/pull/7043

Reply:
I think what you've done here is excellent and fine as-is tho 👍

Reply:
For some reason, office-ui-fabric-react doesn't depend on test-utilities?? Not sure why but don't really want to bother with changing it right now.

Reply:
> 
> 
> For some reason, office-ui-fabric-react doesn't depend on test-utilities?? Not sure why but don't really want to bother with changing it right now.

No worries, I was just sharing for awareness, totally reasonable!